go/vt/vtgate/plugin_mysql_server.go

```diff
@@ -53,6 +53,9 @@ var (
 
 	mysqlSlowConnectWarnThreshold = flag.Duration("mysql_slow_connect_warn_threshold", 0, "Warn if it takes more than the given threshold for a mysql connection to establish")
 
+	mysqlConnReadTimeout  = flag.Duration("mysql_server_read_timeout", 24*time.Hour, "connection read timeout")
```

Comment:
One thing I missed the first time around -- I think this should default to 0 (i.e. unlimited) to preserve current behavior.

Reply:
Good catch. I thought about this and then forgot 😆  - Done.